node/node.go

```diff
@@ -19,7 +19,7 @@ import (
 	"github.com/NethermindEth/juno/l1"
 	"github.com/NethermindEth/juno/metrics"
 	"github.com/NethermindEth/juno/migration"
-	"github.com/NethermindEth/juno/node/http"
+	junohttp "github.com/NethermindEth/juno/node/http"
```

Comment:
`node/http` should be available as `junohttp` already.

Reply:
Can you elaborate more? You mean provide both imports ?
Btw, that import renaming came form Go's tooling.

Reply:
I mean that `github.com/NethermindEth/juno/node/http` should be enough, we don't need the `junohttp` before the import path.

> import renaming came form Go's tooling.

Interesting! Maybe I'm not aware of a best practice. Can you point me to the tool that did that? My LSP/linting setup doesn't complain, and neither does `golangci-lint run`.

Reply:
[Go's official Extension](https://marketplace.visualstudio.com/items?itemName=golang.Go)
I guess the auto renaming import is due to the how the node/http package [was referred](https://github.com/NethermindEth/juno/blob/02e7ebfeaddc7ff9209ed269f8dafaa33e193d07/node/node.go#L101) to